Add Data Studios Kubernetes configuration docs (#83)
seqeralabs/docs: platform_versioned_docs/version-24.1.0/enterprise/docker-compose.mdx

:::note
For more information on configuration, see [Configuration options](./configuration/overview.mdx).
:::

## Optional features

### Data Studios

> Seqera Platform provides an interactive tertiary analysis environment that you can enable for organizational workspaces. For more information enabling Data Studios, see [Data Studios deployment](./enterprise/data-studios.mdx).

Reviewer: ```suggestion
[Data Studios](../data/data-studios.mdx) is an interactive tertiary analysis environment available in organizational workspaces. To enable Data Studios, see [Data Studios deployment](./enterprise/data-studios.mdx).
```

:::note

> Data Studios is currently in **public preview** and is available from Seqera Platform v24.1. If you experience any problems during the deployment process please contact your account executive. Data Studios in Enterprise is not enabled by default. You can enable Data Studios in the [environment variables configuration](./enterprise/configuration/overview#opt-in-seqera-features).

Reviewer: Account executive or support?

Reviewer: Also, enabling Data Studios involves more than just the env var, as it requires the deployment steps referenced above. So this admonition may be misleading.
